src/Fantomas.Core.Tests/ShebangTests.fs

```diff
@@ -0,0 +1,44 @@
+module Fantomas.Core.Tests.ShebangTests
+
+open NUnit.Framework
+open FsUnit
+open Fantomas.Core.Tests.TestHelper
+
+[<Test>]
+let ``should keep a shebang, 2367`` () =
+    let codeSnippet =
+        """#!/usr/bin/env -S dotnet fsi
+// random licensing stuff
+
+open System
+
+printfn "the best thing we've ever done"
+"""
+
+    formatSourceString false codeSnippet config |> should equal codeSnippet
+
+[<Test>]
```

Comment:
Please remove these validation tests, you are testing the parser more than the actual Fantomas code.
`formatSourceString` will also already validate the code.

https://github.com/fsprojects/fantomas/blob/c81e47465df56e4b49e6aab63f25ea9adf56ed90/src/Fantomas.Core.Tests/TestHelpers.fs#L23-L35

Reply:
Done!